docs/pricking.html

```diff
@@ -0,0 +1,476 @@
+                <option value="?tesselace=4x4_117301&patchWidth=12&patchHeight=12&tile=437-,4-73,7-43,734-&shiftColsSW=0&shiftRowsSW=4&shiftColsSE=4&shiftRowsSE=0&">4x4_117301</option>option>
+                <option value="?tesselace=4x4_117397&patchWidth=12&patchHeight=12&tile=437-,4-7-,8-15,8315&shiftColsSW=0&shiftRowsSW=4&shiftColsSE=4&shiftRowsSE=0&">4x4_117397</option>option>
+                <option value="?tesselace=4x4_117415&patchWidth=12&patchHeight=12&tile=437-,4-7-,-5-5,535-&shiftColsSW=0&shiftRowsSW=4&shiftColsSE=4&shiftRowsSE=0&">4x4_117415</option>option>
+            </select>
+            that doesn't fail due to other bugs<br>
+            or try your luck:
+            <ul>
+                <li>Create a custom <a href="/GroundForge/tiles" target="_blank">pattern</a>,
+                    requirements for now:
+                    <br>
+                    no overlapping tile layout, head side and foot side are ignored.
+                    Still, many patterns won't work.
+                </li>
+                <li>
+                    Either follow the <em>W.I.P.</em> link to get the pair diagram,
+                    <br>
+                    or adjust the adress bar to get the thread diagram:
+                    replace <em>tile</em> with <em>pricking</em>.
+                </li>
+            </ul>
+        </li>
+        <li>
+            <strong>Hover</strong> over diagram elements for pop up labels<br>
+            Some browsers show these labels only on the darker elements, clicking fainter objects might still work.
+            <ul>
+                <li>
+                    read pairs between stitches as:
+                    "<em>left/right out of [id], left/right-into [id]</em>".
+                    The optional number at the end is the strength.
+                    These pop ups are collected in the "customized pattern" link.
+                </li>
+                <li>
+                    a stitch shows the <em>[id]</em> that appears on the label of pairs between stitches
+                    and shows the chosen instructions.
+                </li>
+            </ul>
+        </li>
+        <li><strong>Edit</strong> (actions may be slow and have no busy indication)
+            <ul>
+                <li>
+                    Click a pair <span class="disabled">(or not yet implemented: a hole)</span> to make distances between stitches:
+                    <br>
+                    <input type="number" value="50" id="speed" name="speed" step="10" min="10" max="100" onchange="changeSpeed(this)">
+                    <label for="speed">%</label>
+                    <input type="radio" name="weight" id="stronger" value="2">
+                    <label for="stronger">shorter</label>
+                    <input type="radio" name="weight" id="weaker" value="0.5">
+                    <label for="stronger">longer</label>
+                    <input type="radio" name="weight" id="reset-strength" checked value="1">
```

Comment:
![image](https://user-images.githubusercontent.com/10553630/106378653-cf629f00-63a6-11eb-8b1a-3417bf81e909.png)
 
A percentage of the length is something users will understand. But actually it is a percentage of the strength. As long as the strength is 1, it is more or less the same. 

@veronika Is there some formula to compensate? In my opinion it doesn't need to be very exact.

https://github.com/d-bl/GroundForge/blob/14d5e99e9868f6ef430908b475e697132d0e9947/docs/js/pricking.js#L102-L105
https://github.com/d-bl/GroundForge/blob/1633c61ef8638909163ef198b3e09b7c09d2505b/src/main/scala/dibl/fte/TopoLink.scala#L42-L51

Reply:
There is no way to predict what length an edge will have based on its weight.  The system of springs is in an equilibrium.  How the strength of the spring affects that equilibrium is based on a force balancing equation that is quite complex.

Reply:
I meant what the relative change in length would be. But meanwhile I guessed even that depends on the ratio of average weights of (close?) neighbors. Indeed too complicated. 

I changed the wording into "stronger/weaker" but that does not match up with "distances". Any suggestion?

While playing with it I decided to make the lines a little thinner/thicker with the weight, could also make them darker/lighter. The change is subtle, but it doesn't make much sense to make thick lines stronger or thin lines weaker when their neighbors don't have the a similar weight. The pop-ups are a much slower clue.